libnd4j/pom.xml

```diff
@@ -169,38 +169,26 @@
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
                 <version>1.6.0</version>
+
                 <executions>
-                    <!--<execution>-->
-                        <!--<id>libnd4j.cpu.test.clean</id>-->
-                        <!--<phase>clean</phase>-->
-                        <!--<goals>-->
-                            <!--<goal>exec</goal>-->
-                        <!--</goals>-->
-                        <!--<configuration>-->
-                            <!--<executable>make</executable>-->
-                            <!--<workingDirectory>${project.basedir}/tests_cpu</workingDirectory>-->
-                            <!--<arguments>-->
-                                <!--<argument>clean</argument>-->
-                            <!--</arguments>-->
-                        <!--</configuration>-->
-                    <!--</execution>-->
                     <execution>
                         <id>libnd4j.cpu.test.run</id>
                         <phase>test</phase>
                         <goals>
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <skip>${maven.test.skip}</skip>
+                            <skip>${maven.libnd4j.test.skip}</skip>
```

Comment:
If we remove maven.test.skip from here, we should offer some other way to skip all the tests without having to specify 2 or more options on the command line.

Reply:
Got it, let me try to fix it.

Reply:
Wouldn't we be building only for libnd4j in those cases with something like `mvn -f libnd4j` anyway?